lua/entities/starfall_prop/init.lua

```diff
@@ -15,6 +15,18 @@ function ENT:Initialize()
 	self:AddEFlags( EFL_FORCE_CHECK_TRANSMIT )
 end
 
+function ENT:PhysicsSimulate(physObj, dt)
+	if self.customForceMode then
+		return self.customForceAngular, self.customForceLinear, self.customForceMode
+	elseif self.customShadowForce then
+		self.customShadowForce.deltatime = dt
+		physObj:ComputeShadowControl(self.customShadowForce)
+		return SIM_NOTHING
```

Comment:
Should these be `return VECTOR_ZERO, VECTOR_ZERO, SIM_NOTHING`?

Reply:
Looking at how other people use this hook, it looks like it supports returning `SIM_NOTHING` as the only argument as well.  
Cannot find any errors or crashes, dedicated server doesn't complain about physics either when this is used.

Reply:
ah ok cool